script/bootstrap

```diff
@@ -22,7 +22,12 @@ test -d $VENV || python3 -m venv $VENV || return
 python -m pip install -U pip --no-cache-dir
 
 if [[ -z $SKIP_DEPS ]]; then
-    python -m pip install -e . -r dev_requirements.txt
+    if [[ -z $CI ]]; then
+        export PYTHONPATH="./src:$PYTHONPATH"
+    else
+        python -m pip install -e . --no-deps
```

Comment:
What's the rational for not installing syrupy and using `inv test -d` in local dev?

I was under the impression the `-e` ensures `inv test` will use the source code.

Reply:
Hmm looking at the flags yeah it should. I was running into an where my changes were not reflected in the test run after bootstrap, will double check if I can reproduce.

Reply:
I think I just missed this flag when running the install command manually after recreating my venv. Ran it manually since I can't source the bootstrap script in fish shell 🐟 . Tried repro in bash and zsh but it worked as expected, could run `pytest` without needing to export `PYTHONPATH`, same with fish after using `bash ./script/bootstrap`. Will close 👍